packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -325,43 +320,30 @@ export async function getParameterJson(ctx: SolutionContext) {
     throw new Error("Failed to get target environment name from solution context.");
   }
 
-  let parameterFileName, parameterFolderPath, parameterTemplateFilePath;
-  if (isMultiEnvEnabled()) {
-    parameterFileName = parameterFileNameTemplateNew.replace("@envName", ctx.envInfo.envName);
-    parameterFolderPath = path.join(ctx.root, configsFolder);
-    parameterTemplateFilePath = path.join(
-      path.join(ctx.root, templateFolderNew),
-      parameterTemplateFileName
-    );
-  } else {
-    parameterFileName = parameterFileNameTemplate.replace("@envName", ctx.envInfo.envName);
-    parameterFolderPath = path.join(ctx.root, baseFolder, parameterFolder);
-    parameterTemplateFilePath = path.join(parameterFolderPath, parameterTemplateFileName);
-  }
+  const parameterFileName = parameterFileNameTemplate.replace("@envName", ctx.envInfo.envName);
```

Comment:
if multi-env is not enabled, can we get the default envName here?

Reply:
sure, if multi-env is not enabled, ctx.envInfo.envName is "default"
will set default value as "env"